robottelo/common/ssh.py

```diff
@@ -151,6 +151,9 @@ def command(cmd, hostname=None, output_format=None, timeout=None):
         stdout = stdout.read()
         stderr = stderr.read()
 
+    if stderr:
+        stderr = stderr.decode('utf-8')
```

Comment:
This is already done on line 163. See the comment on the line above 163 for more information.